scripts/test/test-coverage.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+./gradlew clean check jacocoTestReport --info
+
+xdg-open build/reports/jacoco/test/html/index.html
+open build/reports/jacoco/test/html/index.html
+start "" build/reports/jacoco/test/html/index.html
```

Comment:
No idea why Jake changed this

Reply:
Dunno mate. That xdg-open is uses file associated to open an application, so I'm guessing that Jake and maybe Rose will have something set up.